### PR TITLE
Update list of repos

### PR DESCRIPTION
**Pull request by mairaw**

### CODE REVIEW

File: Documentation/core-repos.md [hunk 1]
```diff
@@ -59,6 +59,15 @@
 |[dotnet/EntityFramework.Docs](https://github.com/dotnet/EntityFramework.Docs)|EF Core & EF 6 documentation          |
 |[dotnet/sqlclient](https://github.com/dotnet/sqlclient)                      |SqlClient data provider for SQL Server|
 
+## Machine Learning and AI
+
+|Repository                                                                                    |Description              |
+|----------------------------------------------------------------------------------------------|-------------------------|
+|[dotnet/machinelearning](https://github.com/dotnet/machinelearning)                           |ML.NET                   |
+|[dotnet/machinelearning-modelbuilder](https://github.com/dotnet/machinelearning-modelbuilder) |Model Builder            |
```
Comment (terrajobst) [inline Documentation/core-repos.md:67]: That's what it is, rigth?

```suggestion
|[dotnet/machinelearning-modelbuilder](https://github.com/dotnet/machinelearning-modelbuilder) |ML.NET Model Builder UI for Visual Studio|
```

Comment (mairaw) [reply Documentation/core-repos.md:67]: I'll let @briacht comment on the correct description

Comment (Shahidzada) [reply Documentation/core-repos.md]: History

Comment (Shahidzada) [reply Documentation/core-repos.md]: Shahidzada654@gmail.com

**Review by terrajobst — APPROVED** [review]

**Pull request merged**